keepers/validator-keeper/src/main.rs

```diff
@@ -369,5 +406,12 @@ async fn main() {
         args.interval,
     ));
 
+    tokio::spawn(mev_earned_loop(
+        Arc::clone(&client),
```

Comment:
this can be client.clone()

Reply:
done, modified in the existing code as well